node/Cargo.toml

```diff
@@ -148,6 +152,13 @@ runtime-benchmarks = [
     "polkadot-service?/runtime-benchmarks",
     "zeitgeist-runtime/runtime-benchmarks"
 ]
+cli = [
+    'try-runtime-cli',
+]
```

Comment:
We don't need that feature, it's not used anywhere.
```suggestion
```

Reply:
So you mean, that nowhere in the code we use `feature = "cli"`, I understand. So later, we can't use the command line interface for the storage migrations? I don't really get what this feature could be needed apart from that then. Why would the tutorial (https://docs.substrate.io/how-to-guides/v3/tools/try-runtime/) include this then?

Reply:
The tutorial has taken the code from the [substrate node](https://github.com/paritytech/substrate/blob/86b1176d3ba2bd49b87e99e1b074bc0279849021/bin/node/cli/src/lib.rs), which uses a feature `cli` to build the client (otherwise the code will only offer means to create all necessary structures to build your own client). In our case, the client is build by default, there is no feature to toggle building it. That's why we don't need that feature. Our client will always be built and offer the complete try-runtime functionality in case it is built with the `try-runtime` feature.
You can see that the [substrate node template](https://github.com/paritytech/substrate/blob/86b1176d3ba2bd49b87e99e1b074bc0279849021/bin/node-template/node/Cargo.toml#L68-L73) does it the same way, always build the client.